src/typings/models/product.d.ts

```diff
@@ -4,6 +4,7 @@ declare module Models.Product {
     link: string;
     name: string;
     price: string;
+    discountPrice: string;
```

Comment:
change this to discountPrice?

Reply:
I need more explanation on this one.

Reply:
it is nullable value. Maybe we will have a discount price, maybe not. Adding a question to the name will also accept undefined as a value